include/cuco/detail/storage.cuh

```diff
@@ -191,21 +192,21 @@ class aos_storage_ref {
    * @brief Gets windows array.
    *
    * @return Pointer to the first window
-   */
+   */ // TODO should be named data()
```

Comment:
If we go with `data`, the storage ref will be exclusively an array of windows aka `aow_storage_ref`.